content/docs/details/scaffolder/index.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Scaffolder
+publishedDate: '2022-05-13T10:53:00.0Z'
+description: Initialize projects and trigger automation from within Backstage. You can prompt parameter inputs and combine scaffolder actions in order to build a scaffolder template to run within Backstage.
+---
+
+# Overview
+Imagine you’re an engineer looking to create a new microservice. You want to get started as quickly as possible, with minimal boilerplate and red-tape to jump through. At the same time, engineering organizations benefit from having consistency in production, and often put gates in place to enforce it.
```

Comment:
I'd probably be a bit more literal about what the scaffolder does here. This reads like a sales pitch rather than technical documentation.

Reply:
Sure. I copied it from the pitch :D